tensorflow/core/ops/io_ops.cc

```diff
@@ -448,6 +448,7 @@ REGISTER_OP("FixedLengthRecordReader")
     .Attr("record_bytes: int")
     .Attr("footer_bytes: int = 0")
     .Attr("hop_bytes: int = 0")
+    .Attr("encoding: string = ''")
```

Comment:
There should be no new uses of the V1 op, so there should be no reason to change it.